README.md

```diff
@@ -111,6 +111,7 @@ run-time behavior:
   until wastebin responds with 408, by default it is set to 5 seconds.
 * `WASTEBIN_MAX_BODY_SIZE` number of bytes to accept for POST requests. Defaults
   to 1 MB.
+* `WASTEBIN_MAX_PASTE_EXPIRY` maximum allowed lifetime of a paste. Leave empty or set to -1 for no limit. Defaults to no limit.
```

Comment:
Not to start any geopolitics but for some reason I'd prefer "expiration" over "expiry" but if you have good arguments we could keep it.

Reply:
I do not lean either way on this so im fine with `expiration`